examples/app/src/BaselineInputs.tsx

```diff
@@ -106,31 +106,29 @@ export const BaselineInputs = () => {
 
           <div className="row mt-4">
             <div className="col">
-              <FieldSet legend="Checkbox List">
-                <CheckboxList
-                  name="checkboxList"
-                  options={[
-                    { label: "Foo", value: "foo" },
-                    { label: "Bar", value: "bar" },
-                    { label: "Disabled", value: "buzz", disabled: true }
-                  ]}
-                  {...fields.checkboxList}
-                />
-              </FieldSet>
+              <CheckboxList
+                title="Checkbox List"
```

Comment:
Should this be legend? That way, we can pass it right through to the FieldSet component.